DeviceAdministration/Web/App_Start/Startup.Autofac.cs

```diff
@@ -88,6 +88,7 @@ protected override void Load(ContainerBuilder builder)
             builder.RegisterType<CellularExtensions>().As<ICellularExtensions>();
             builder.RegisterType<AzureTableStorageClientFactory>().As<IAzureTableStorageClientFactory>();
             builder.RegisterType<BlobStorageClientFactory>().As<IBlobStorageClientFactory>();
+            builder.RegisterGeneric(typeof(DocumentDBClient<>)).As(typeof(IDocumentDBClient<>));
```

Comment:
@kapil-ms When registering open generics, like DocumentDBClient<>, you cannot use a generic type reference. Eg. RegisterGeneric<DocumentDBClient<>> doesn't work as the compiler and runtime does not support it. For open generics registration is always done via typeof(x). Also that's how Autofac requires open generic registration.